processor.cpp

```diff
@@ -4899,9 +4883,9 @@ bool field::process(Processors::Adjust& arg) {
 		}
 		if(pos_adjust.size()) {
 			core.re_adjust = true;
-			group* ng = pduel->new_group();
+			auto ng = pduel->new_group();
 			ng->container.swap(pos_adjust);
-			ng->is_readonly = TRUE;
+			ng->is_readonly = true;
```

Comment:
all of these make me think that you should be able to set the readonly flag directly within the ctor, but anyways that shouldn't be the focus of this PR.

Reply:
Agree with that, but as you said it's a different matter